test/test-server/hub_router.go

```diff
@@ -0,0 +1,18 @@
+package testserver
+
+import (
+	"testing"
+
+	"github.com/gorilla/mux"
+)
+
+func NewHubRouter(t *testing.T) *mux.Router {
+	router := mux.NewRouter()
+	router.Use(defaultHeaderMiddleware)
+
+	router.HandleFunc("/api/plugins/{owner}/{id}", handleHubPlugin(t))
```

Comment:
Can we match the style of our other tests? We're using a list or map for all of our routes (I forget which).